packages/core/src/components/overflow-list/overflow-list.md

```diff
@@ -1,6 +1,19 @@
 @# Overflow list
 
-`OverflowList` displays a list of items that can overflow on its own. The required `visibleItemRenderer` callback prop allows for customizing the appearance of visible items. The required `overflowRenderer` callback prop allows for customizing the rendering of collapsed items. 
+`OverflowList` takes a generic list of items and renders as many items as can
+fit inside itself. Other items are collapsed into an overflow menu. The visible
+items will be recomputed when a resize is detected.
+
+The `items` prop accepts an array of generic objects. The required
+`visibleItemRenderer` callback prop determines the appearance of a visible item.
+The required `overflowRenderer` callback prop receives all overflowed items
+and renders the overflow menu or indicator.
```

Comment:
`and allows customizing the rendering of overflown items`?

Reply:
it's not custom though, it's required

Reply:
That's not what I mean. The prop is required yes, but rendering an overflow menu or indicator isn't required. I can render a menu, an static icon, or absolutely nothing I want to--that's what's powerful